corehq/apps/es/tests/test_user_es.py

```diff
@@ -1,61 +0,0 @@
-from django.test import TestCase
-
-from corehq.apps.domain.shortcuts import create_domain
-from corehq.apps.es import UserES
-from corehq.apps.es.users import user_adapter
-from corehq.apps.es.client import manager
-from corehq.apps.es.tests.utils import es_test
-from corehq.apps.users.dbaccessors import delete_all_users
-from corehq.apps.users.models import CommCareUser
-from corehq.util.es.testing import sync_users_to_es
-
-
-@es_test(requires=[user_adapter], setup_class=True)
-class TestUserES(TestCase):
```

Comment:
This test class tests the `user_data` function, but after this PR, there's only one use of that, and it's well tested elsewhere